rllib/algorithms/sac/sac_rl_module.py

```diff
@@ -0,0 +1,106 @@
+from typing import Type
+from ray.rllib.algorithms.sac.sac_catalog import SACCatalog
+
+# from ray.rllib.algorithms.sac.sac_learner import QF_PREDS
+# from ray.rllib.core.models.base import ACTOR, CRITIC
+from ray.rllib.core.models.specs.typing import SpecType
+from ray.rllib.core.rl_module.rl_module import RLModule
+from ray.rllib.core.rl_module.rl_module_with_target_networks_interface import (
+    RLModuleWithTargetNetworksInterface,
+)
+from ray.rllib.models.distributions import Distribution
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import ExperimentalAPI, override
+
+CRITIC_TARGET = "critic_target"
+QF_PREDS = "qf_preds"
+
+
+@ExperimentalAPI
+class SACRLModule(RLModule, RLModuleWithTargetNetworksInterface):
+    def setup(self):
```

Comment:
Add `override` decorator.

Question: Should we call `super` here or not?

Reply:
We should try to be super consistent and clear about the two decorators:

`@OverrideToImplementCustomLogic`
and 
`@OverrideToImplementCustomLogic_CallToSuperRecommended`

Can you check, whether these already exist in the base: `RLModule.setup()` and if not add them as applicable?

Reply:
Both are used in the super class. I think I remember that I ran into an error when calling ` super().setup()`. I have to recheck, when testing.